src/main/java/com/github/tombentley/krpccodegen/main/KrpcGenerator.java

```diff
@@ -208,9 +219,7 @@ private Configuration buildFmConfiguration() throws IOException {
         Version version = Configuration.VERSION_2_3_31;
         Configuration cfg = new Configuration(version);
 
-        // Specify the source where the template files come from. Here I set a
-        // plain directory for it, but non-file-system sources are possible too:
-        cfg.setDirectoryForTemplateLoading(templateDir);
+        cfg.setClassLoaderForTemplateLoading(templateClassLoader, "/");
```

Comment:
Do we _really_ want to load templates via a class loader? Doesn't it imply that they have to be in a jar, rather than just files on the filesystem? I would have thought that it would be better to move the `Filter.ftl` to kproxy and have the maven plugin just look in `src/main/krpc-templates` in the kproxy repo.

Reply:
Very good question. In fact my thinking was that those templates are rather "static" and thus should be shipped via the plug-in (in which case we need to look them up via the plug-ins class loader). But you may be right and they are usage specific, in which case moving them over to kproxy makes absolute sense.